tests/manage/test_pvc_deletion_during_io.py

```diff
@@ -4,7 +4,7 @@
 from ocs import defaults, exceptions, constants
 from resources import pod
 from ocsci.testlib import ManageTest, tier1
-from tests.fixtures import (
+from tests.conftest import (
```

Comment:
You shouldn't need to import fixtures. The ``@pytest.mark.usefixtures`` call below could just pass the name of the fixture as a string. For example:

```
@pytest.mark.usefixtures(
    'create_rbd_secret',
    'create_ceph_block_pool',
    ...
)
```

Here's a link to the docs with more info on that: https://docs.pytest.org/en/latest/fixture.html#using-fixtures-from-classes-modules-or-projects

Reply:
As @andrewschoen already noted, this is not expected way to use fixtures.

Reply:
Great input guys!
Will fix accordingly